smart-contracts/contracts/StrollManager.sol

```diff
@@ -1,56 +1,21 @@
 //SPDX-License-Identifier: Unlicense
-pragma solidity ^0.8.4;
+pragma solidity =0.8.13;
```

Comment:
is enough to declare 0.8.13

Reply:
Isn't this a bit more precise though? Have seen this used somewhere.